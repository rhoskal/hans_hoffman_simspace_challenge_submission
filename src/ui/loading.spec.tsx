import React from "react";
import { render } from "@testing-library/react";

import { Loading } from "./loading";

describe("[Component]: Loading", function() {
  it("should match snapshot", function() {
    const { container } = render(<Loading />);

    expect(container).toMatchSnapshot();
  });
});
