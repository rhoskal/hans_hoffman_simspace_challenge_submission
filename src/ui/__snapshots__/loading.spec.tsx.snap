// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: Loading should match snapshot 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
